test/support/live_views/elements.ex

```diff
@@ -104,17 +104,29 @@ defmodule Phoenix.LiveViewTest.ElementsLive do
       <input name="hello[utc_text]" type="text">
       <%= Phoenix.HTML.Form.datetime_select :hello, :utc_select, second: [] %>
     </form>
+
+    <form id="trigger-form" phx-submit="form-submit-trigger" phx-value-key="value" phx-trigger-action=<%= @trigger_action %>>
+    </form>
     """
   end
 
   def mount(_params, _session, socket) do
-    {:ok, assign(socket, :event, nil)}
+    socket =
```

Comment:
Not to bikeshed, but I don't know if this is idiomatic or not. I haven't looked at many others' liveview code.